capnproto-java/meta.yaml

```diff
@@ -21,11 +21,10 @@ build:
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - capnproto
```

Comment:
this is a little bit surprising

Reply:
It stays in the `host` section so it's not so surprising. `host` and `build` are both used to build packages, the distinction is mostly to allow cross-platform building and apart from that the convention is that `build` should only contain build tools and `host` the rest like libraries that are used by the linker etc. Placing the same package in both of these sections is almost always redundant.